python-project-template/docs/Makefile

```diff
@@ -1,18 +1,28 @@
-# Minimal makefile for Sphinx documentation
+# Makefile for Sphinx documentation
 #
 
 # You can set these variables from the command line, and also
 # from the environment for the first two.
 SPHINXOPTS    ?= -T -E -d _build/doctrees -D language=en
+EXCLUDENB 	  ?= -D exclude_patterns="notebooks/*"
 SPHINXBUILD   ?= sphinx-build
 SOURCEDIR     = .
 BUILDDIR      = ../_readthedocs/
 
+.PHONY: help no-nb no-notebooks clean Makefile
```

Comment:
I am used to seeing .PHONY at the end of the Makefile but I do not know if it matters. Its sole purpose is to declare targets that do not represent actual files. So I do not think that Makefile needs to be present (unless you noticed some error?)

Reply:
The initial Makefile that Sphinx generates has the line `.PHONY: help Makefile`, so I reasoned it was best keep it if I had no specific reason to remove it.

[Based on this SO](https://stackoverflow.com/questions/43967578/default-makefile-for-sphinx), it seemed to me like it's there to make sure whatever targets are captured by `%: Makefile` are always run, and--this part goes a little over my head--the presence of `Makefile` as a dependency of `%` to avoid [weird GNU Make behavior](https://stackoverflow.com/questions/40062551/a-surprising-behaviour-of-gnu-make-when-using-as-target/40062635#40062635).